packages/server/modules/workspaces/tests/integration/repositories.spec.ts

```diff
@@ -0,0 +1,86 @@
+// TODO: test deleting user doesn't delete the workspace they created
```

Comment:
todo

Reply:
This can be removed, I deleted the user foreign key from the workspace table